### ISSUE TITLE
No speed improvements after TF-TRT optimizing on a tensorflow BERT model

### ISSUE THREAD

**Issue by SohaKhazaeli** [NVIDIA/TensorRT#2480]
## Description

After optimizing the model with either FP32 or FP16 I don't get any speed improvements.

## Environment
The optimization is done on tensorflow/tensorflow:2.10.0-gpu docker image. The model uses tensorflow-text and tf-models-official libraries

**NVIDIA GPU**: T4
**CUDA Version**:  11.2.1
**CUDNN Version**:  8.1.0.77-1
**Python Version (if applicable)**: 3.9
**Tensorflow Version (if applicable)**: 2.10
**docker**: tensorflow/tensorflow:2.10.0-gpu

## Relevant Files

![image](https://user-images.githubusercontent.com/38143776/201760685-bfc6bb6c-d1e3-4d4b-a0ac-70de01f1651d.png)

**Comment by zerollzeng** [comment]
There is huge framework overhead with TF-TRT, can you try onnx-tensorrt?

**Comment by ncomly-nvidia** [comment]
please file TF-TRT issues to https://github.com/tensorflow/tensorrt

**Comment by SohaKhazaeli** [comment]
@zerollzeng My model is a tf model. Do you recommend to recommend it to onnx and the use onnx-tensorrt?
@ncomly-nvidia I did file TF-TRT issues to https://github.com/tensorflow/tensorrt 14 days ago. I didn't get any response.

**Comment by zerollzeng** [comment]
> @zerollzeng My model is a tf model. Do you recommend to convert it to onnx and then use onnx-tensorrt?

Yes, it can help us narrow down the issue to see whether this is a trt issue or not. as I said, TF-TRT has framework overhead.
